src/pages/workspaces/workspace/Container.js

```diff
@@ -128,7 +134,11 @@ const WorkspaceContainer = hh(class WorkspaceContainer extends Component {
           ]
         )
       ]),
-      tabComponents[activeTab]({ namespace, name, key: forceUpdateKey })
+      Utils.cond(
+        [workspaceFailure, ''],
```

Comment:
No error message? I still think we need a more general solution for this—I see fatigue here from constantly having to handle this case.